### DISCUSSION TITLE
How to incorporate a new template file (not an override)

### DISCUSSION THREAD

**Discussion by ben519** [squidfunk/mkdocs-material#3456] [Q&A] [answered]
I'd like to implement a 401.html page into the root of my site. Similar to [404.html](https://github.com/squidfunk/mkdocs-material/blob/master/material/404.html), I'd expect the template to look like this

**_401.html_**
```
{% extends "main.html" %}
{% block content %}
  <h1>401 - Unauthorized</h1>
{% endblock %}
```

If I put this in `/overrides/` it doesn't get picked up by the build, I assume because it's not actually overriding an existing template.

Is there a way to make this work? Alternatively, if I make a PR for this, would you be open to merging it? Thanks!

**Comment by squidfunk** [comment, marked as answer]
If you list them under [`static_templates`](https://www.mkdocs.org/dev-guide/themes/#static_templates), they will be copied during the build:

https://github.com/squidfunk/mkdocs-material/blob/a8c5da8a351e5c8b89959cbfccc1171dbc7810b9/mkdocs.yml#L42-L44

**Reply by ben519** [reply]
Not sure how I missed this. Thank you so much for your help.
